clang/test/dpct/query_api_mapping/Runtime/test_graphics_interop_d3d11_lin.cu

```diff
@@ -0,0 +1,11 @@
+// UNSUPPORTED: system-windows
```

Comment:
line 2 is used on linux version cuda header, line 3 is used on windows version cuda header. So when the case cannot support windows, we can remove line 3.